README.md

```diff
@@ -13,10 +13,11 @@
 ## Element architecture
 
 In both of the following diagrams, the trial table starts immediately downstream from
-***Session***. In one case, Sessions are first segmented into trials, and then 
-segmented into events. This might be appropriate, for example, in a paradigm with 
-repeated conditions and response behaviors associated with different conditions. In the 
-next, Sessions are directly upstream from both Trials and Events. This might be appropriate for a paradigm that recorded events within naturalistic free behavior. We  provide an
+***Session***. In one case, Sessions are first segmented into trials, and then segmented
+into events. This might be appropriate, for example, in a paradigm with repeated
+conditions and response behaviors associated with different conditions. In the next,
+Sessions are directly upstream from both Trials and Events. This might be appropriate
+for a paradigm that recorded events within naturalistic free behavior. We provide an
```

Comment:
Please rephrase as there is only 1 diagram.

Reply:
All set